scRNA-seq/README.md

```diff
@@ -7,19 +7,16 @@ It covers normalization and dimension reduction methods that can be used for bot
 
 The notebooks that comprise this module are:
 
-* [Quantification of tag-based scRNA-seq data](https://alexslemonade.github.io/training-modules/scRNA-seq/01-scRNA_quant_qc.nb.html)
-* [Quality control and filtering ](https://alexslemonade.github.io/training-modules/scRNA-seq/02-filtering_scRNA.nb.html)
-* [Normalization](https://alexslemonade.github.io/training-modules/scRNA-seq/03-normalizing_scRNA.nb.html)
-* [Dimension reduction](https://alexslemonade.github.io/training-modules/scRNA-seq/04-dimension_reduction_scRNA.nb.html) 
-* [Clustering](https://alexslemonade.github.io/training-modules/scRNA-seq/05-clustering_markers_scRNA.nb.html)
+- [Quantification of tag-based scRNA-seq data](https://alexslemonade.github.io/training-modules/scRNA-seq/01-scRNA_quant_qc.nb.html)
+- [Quality control and filtering ](https://alexslemonade.github.io/training-modules/scRNA-seq/02-filtering_scRNA.nb.html)
+- [Normalization](https://alexslemonade.github.io/training-modules/scRNA-seq/03-normalizing_scRNA.nb.html)
+- [Dimension reduction](https://alexslemonade.github.io/training-modules/scRNA-seq/04-dimension_reduction_scRNA.nb.html) 
+- [Clustering](https://alexslemonade.github.io/training-modules/scRNA-seq/05-clustering_markers_scRNA.nb.html)
+- [Cell type identification](https://alexslemonade.github.io/training-modules/scRNA-seq/06-celltype_assignment.nb.html)
```

Comment:
Make sure this making scheme is consistent. Previously was written as "cell type _annotation_", so pick your favorite!